src/onboarding/restore-backup/onboarding-restore-backup-restore.ts

```diff
@@ -35,131 +34,280 @@ class OnboardingRestoreBackupRestore extends LitElement {
 
   @state() private _loading = false;
 
+  @state() private _selectedData?: BackupData;
+
+  @query("ha-progress-button")
+  private _progressButtonElement!: HaProgressButton;
+
   render() {
-    const agentId = this.supervisor ? HASSIO_LOCAL_AGENT : CORE_LOCAL_AGENT;
+    const agentId = Object.keys(this.backup.agents)[0];
     const backupProtected = this.backup.agents[agentId].protected;
 
     return html`
+      <ha-icon-button-arrow-prev
+        .label=${this.localize("ui.panel.page-onboarding.restore.back")}
+        @click=${this._back}
+      ></ha-icon-button-arrow-prev>
+      <h1>
+        ${this.localize(
+          "ui.panel.page-onboarding.restore.details.restore.title"
+        )}
+      </h1>
+
+      ${this.backup.homeassistant_included
+        ? html`<div class="description">
+            ${this.localize(
+              "ui.panel.page-onboarding.restore.confirm_restore_full_backup_text"
+            )}
+          </div>`
+        : html`
+            <ha-alert alert-type="error">
+              ${this.localize(
+                "ui.panel.page-onboarding.restore.details.home_assistant_missing"
```

Comment:
Why do we show the rest of this page interactive if it isnt going to work anyway?